pages/en/lb4/Application.md

```diff
@@ -47,17 +51,6 @@ export class WidgetApplication extends Application {
     app.controller(DoohickeyController);
   }
 
-  async start() {
-    // This is where you would asynchronously retrieve servers, providers and
-    // other components to configure them before launch.
-    const server = await app.getServer(RestServer);
-    server.bind('rest.port').to(8080);
-    server.api(WidgetApi);
```

Comment:
This is a breaking change in the context of the original example. The original example assumed api-first approach, where `WidgetApi` contained Swagger spec describing REST API, with extensions like `x-controller-name` and `x-method` used to link Swagger endpoints with controller method implementing them.

Since api-first support is out of scope of our upcoming release, I am ok to rework this example to code-first approach, where the REST API is described on controller classes/methods using decorators like `@get` and `@param`.

In other words, this change is fine with me too 👍